crates/erg_parser/parse.rs

```diff
@@ -1733,10 +2147,21 @@ impl Parser {
                     .map_err(|_| self.stack_dec(fn_name!()))?;
                 if let Some(tk) = self.peek() {
                     if tk.is(Mutate) {
-                        let err = ParseError::invalid_mutable_symbol(
+                        self.skip();
+                        let main_msg = switch_lang!(
+                            "japanese" => "不正な可変定義です",
```

Comment:
Good support, but `The mutation operator is a prefix operator, not a postfix operator` may be a better description.